packages/frameworks-astro/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@auth/astro",
+  "description": "Authentication for Astro.",
+  "version": "0.1.0",
+  "type": "module",
+  "files": [
+    "api",
+    "src",
+    "middleware.ts",
+    "module.d.ts"
+  ],
+  "exports": {
+    ".": "./src/index.ts",
+    "./config": "./src/config.ts",
+    "./client": "./src/client.ts",
+    "./server": "./src/server.ts",
+    "./middleware": "./middleware.ts"
```

Comment:
```suggestion
    ".": {
      "types": "./index.d.ts",
      "import": "./index.js"
    },
    "./config": {
      "types": "./config.d.ts",
      "import": "./config.js"
    },
    "./client": {
      "types": "./client.d.ts",
      "import": "./client.js"
    },
    "./server": {
      "types": "./server.d.ts",
      "import": "./server.js"
    },
    "./middleware": {
      "types": "./middleware.d.ts",
      "import": "./middleware.js"
    },
    "./adapters": {
        "types": "./adapters.d.ts"
      },
    "./providers/*": {
      "types": "./providers/*.d.ts",
      "import": "./providers/*.js"
    }
```

Reply:
The Astro package doesn't require bundling, as all Astro projects (regardless of config) support TypeScript files. Is there still a reason to specify both `types` and `import` if they both lead to the same file?